### ISSUE TITLE
Object detection and box tracking in python

### ISSUE THREAD

**Issue by HungDuong1998** [google/mediapipe#2632]
<em>Please make sure that this is a [solution](https://google.github.io/mediapipe/solutions/solutions.html) issue.<em>

**System information** (Please provide as much relevant information as possible)
- Have I written custom code (as opposed to using a stock example script provided in Mediapipe): Yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04, Android 11, iOS 14.4): Windows 10
- [MediaPipe version](https://github.com/google/mediapipe/releases): 0.8.7.3
- Bazel version: 4.2.1
- Solution (e.g. FaceMesh, Pose, Holistic): Object Tracking
- Programming Language and version ( e.g. C++, Python, Java): Python

**Describe the expected behavior:**

I'm trying to build Object Tracking solution in Python following your example graph in [tracking graph](https://github.com/google/mediapipe/tree/master/mediapipe/graphs/tracking) section, copying the [object detection graph for desktop](https://github.com/google/mediapipe/blob/master/mediapipe/graphs/tracking/object_detection_tracking_desktop_live.pbtxt)

**Standalone code you may have used to try to get what you need :**

This is my graph, the detection part works on following this [forked version of object detection for python](https://github.com/aniketiq/mediapipe/blob/ObjectDetectionSolution/mediapipe/python/solutions/object_detection.py), meaning that the PacketResamplerCalculator is working properly. But on using the tracking part, however, my ObjectTrackingSubgraph doesn't seem to work. I don't know is it because of the native calculator or because of the BoxTrackingSubgraph module.

If there is a problem, provide a reproducible test case that is the bare minimum necessary to generate the problem. If possible, please share a link to Colab/repo link /any notebook:

Object Detection Tracking graph: 
![image](https://user-images.githubusercontent.com/87010829/135954806-af8412d2-7f5f-4d20-b32e-50e999ae17cc.png)

Object Tracking Sub Graph:
![image](https://user-images.githubusercontent.com/87010829/135954838-b1915ffa-35f0-4d7c-895e-f929f2823c3b.png)

Box Tracking Sub Sub Graph: :D
![image](https://user-images.githubusercontent.com/87010829/135954877-bd820a68-4b77-4ff1-9e54-77d9b7cabb51.png)

**Other info / Complete Logs :**
Include any logs or source code that would be helpful to
diagnose the problem. If including tracebacks, please include the full
traceback. Large logs and files should be attached:
Bazel built successfully but the result outputs in Python API are showing None

**Comment by sgowroji** [comment]
Hi @HungDuong1998,  What is the error while running ObjectTrackingSubgraph. Could you please share the code changes and logs to help understand the issue. Thanks!

**Comment by HungDuong1998** [comment]
It does not show syntax or library missing errors on running the python API built from setup.py bdist_wheel, the build was successful. 
However the results are not as expected. When I try to run the [live demo of object tracking from native c++ code](https://github.com/google/mediapipe/tree/master/mediapipe/examples/desktop/object_tracking) it works well as my expectation. But on copying the [example graphs](https://github.com/google/mediapipe/blob/master/mediapipe/graphs/tracking/object_detection_tracking_desktop_live.pbtxt) to use in python API with `setup.py gen_protos && setup.py bdist_wheel + pip install dist/mediapipe.whl` it seems that the ObjectTrackingSubgraph does not run like the demo anymore, the results showing only None. I'm suspecting that direct copy and paste does not work on python API no?
Many thanks

**Comment by HungDuong1998** [comment]
@sgowroji this is my [forked version](https://github.com/HungDuong1998/mediapipe) showing the changes log, please take a look

**Comment by HungDuong1998** [comment]
Update: I have successfully deployed the TrackingSubGraph, turned out that I did not understand the console log. It keeps saying 
> no history. confidence: 1
making me confused about the history it was mentioning.

I will close the issue now. [My forked version](https://github.com/HungDuong1998/mediapipe) has successfully deployed object box tracking and detection in python API for anyone interested. (CPU version)

Steps to build:
1: Follow [google official guide on installing mediapipe for windows](https://google.github.io/mediapipe/getting_started/install.html#installing-on-windows) using my forked version
2: From the root directory run `setup.py gen_protos && setup.py bdist_wheel`
3: From your python environment run `pip install dist/mediapipe_generated_wheel.whl`

Many thanks all.
